lms/djangoapps/email_marketing/signals.py

```diff
@@ -213,6 +214,10 @@ def _create_sailthru_user_vars(user, profile, registration=None):
         sailthru_vars['activation_key'] = registration.activation_key
         sailthru_vars['signupNumber'] = randint(0, 9)
 
+    enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=_get_current_site(), user=user)
+    if enterprise_learner_data:
```

Comment:
I'll let the Enterprise team review for approach.  I just had a minor coding comment.

Rather than an if-statement, could we instead just always appropriately set the value?
```
sailthru_vars['is_enterprise_learner'] = bool(enterprise_learner_data)
```

Reply:
Actually, would it make sense to encapsulate this (if it hasn't already been done) to something like:
```
sailthru_vars['is_enterprise_learner'] = enterprise_api.is_enterprise_learner(user)
```
I'll let @douglashall speak to this though. :)

Reply:
@robrap I have done the first part, I'll leave second on @douglashall.